quattor/types/metadata.pan

```diff
@@ -4,6 +4,10 @@ type structure_branch = {
     "author" ? string
     "name" : string
     "type" : string
+    @{a unique id, e.g. hash of last git commit}
+    "id" ? string
```

Comment:
wrt to actual usage, i have no idea how aquilon inserts this metadata, but it's the same way i guess.
we use https://github.com/stdweird/quattor-SCDB-ugent/blob/master/bc#L148 to generate the commanline data and then inject it in the compilation using https://github.com/stdweird/quattor-SCDB-ugent/blob/master/bc#L218
the templates have a dummy default value so one can compile and check, the `bc` script is probably equivalent to `aq deploy` and is used on our deploy hosts.